Automattic/wp-calypso
# Remove LUX usage

#### Changes proposed in this Pull Request

* Remove LUX (partially reverts #41539)

#### Testing instructions

* Verify unit test passes
* Checkout the branch and open Calypso:
    * Verify there are no request to `speedcurve.com`
    * Verify the global 'LUX' is not present
